doc/contrib/documentation.md

```diff
@@ -0,0 +1,84 @@
+# Documentation
+
+This document gives guidelines for contributors concerning Elektra's documentation.
+This document takes preference to the actual situation, so if you see documentation
+not according to this document, please [create an issue](https://issue.libelektra.org)
+or directly fix it with your next PR.
+
+> Note: It is always allowed to improve the documentation, in every PR, even if
+> the documentation fix is completely unrelated.
+
+## Target Groups
+
+We write documentation for three groups:
+
+- User:
+  Is a person who uses some tool of Elektra, e.g. `kdb`, Qt-Gui or Web-UI.
```

Comment:
If think this can be broader. Using a tool should not be required. I'd also say using any application which is built with Elektra is enough. For example, even if I don't use `kdb` and instead edit a config file of such an application, I need to know a bit about how Elektra works (keys, etc.)

Reply:
Especially you should know not to directly edit config files but to use `kdb editor` instead :wink: